client/containers/Pub/PubDocument/Review/ReviewerDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Button, Classes, Dialog, InputGroup } from '@blueprintjs/core';
+
+require('./reviewerDialog.scss');
+
+type Props = {
```

Comment:
Please do a pass on these to provide props that are more specific than `any` in each case.